modular_skyrat/modules/customization/__DEFINES/mobs.dm

```diff
@@ -0,0 +1,3 @@
+#define DEFAULT_QUICK_COOLDOWN  4
```

Comment:
Unused define?

Reply:
yes, actually, let me delete and remove that